includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,219 @@
+	 */
+	public function register_routes() {
+		$this->register_get_data_route();
+	}
+
+	/**
+	 * Check user permission for REST API access.
+	 *
+	 * @return bool Whether the user can access the Sensei Home REST API.
+	 */
+	public function can_user_access_rest_api() {
+		return current_user_can( 'manage_sensei' );
+	}
+
+	/**
+	 * Register GET / endpoint.
+	 */
+	public function register_get_data_route() {
+		register_rest_route(
+			$this->namespace,
+			$this->rest_base,
+			[
+				[
+					'methods'             => WP_REST_Server::READABLE,
+					'callback'            => [ $this, 'get_data' ],
+					'permission_callback' => [ $this, 'can_user_access_rest_api' ],
+				],
+			]
+		);
+	}
+
+	/**
+	 * Get data for Sensei Home frontend.
+	 *
+	 * @return array Setup Wizard data
+	 */
+	public function get_data() {
+
+		return [
+			'tasks_list'  => [
+				'tasks' => [
+					// TODO: Generate based on Setup Wizard data + site info.
+					[
+						'title' => 'Set up Course Site',
+						'done'  => true,
+						'url'   => null,
+					],
+					[
+						'title' => 'Create your first Course',
+						'done'  => false,
+						'url'   => '/wp-admin/edit.php?post_type=course',
+					],
+					[
+						'title' => 'Configure Learning Mode',
+						'done'  => false,
+						'url'   => '/wp-admin/edit.php?post_type=course&page=sensei-settings#course-settings',
+					],
+					[
+						'title' => 'Publish your first Course',
+						'done'  => false,
+						'url'   => '???',
+					],
+				],
+				'image' => 'http://...', // For the moment we are returning the task_list image url from the backend. We could see how to calculate this on frontend based on tasks response.
+			],
+			'quick_links' => [
+				// TODO: Generate or hardcode in frontend.
+				'Courses'           => [
+					[
+						'title' => 'Getting Started Guide',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Create a Course',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Import a Course',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Reports',
+						'url'   => 'http://...',
+					],
+				],
+				'Settings'          => [
+					[
+						'title' => 'Email notifications',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Learning Mode',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'WooCommerce',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Content Drip',
+						'url'   => 'http://...',
+					],
+				],
+				'Advanced Features' => [
+					[
+						'title' => 'Interactive Blocks',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Groups & Cohorts',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Quizzes',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'Integrations',
+						'url'   => 'http://...',
+					],
+				],
+			],
+			'help'        => [
+				// TODO: Generate or hardcode in frontend.
```

Comment:
I'm not sure about hardcoding this on the frontend, as we'll need to verify whether the license is valid first. May as well leave it dynamic and coming from the backend, as Jake pointed out.

Reply:
Agree! Added skeleton in 145cee198.